collector/stats_channel.go

```diff
@@ -59,6 +59,22 @@ func ChannelStats(namespace string) StatsCollector {
 			}, labels),
 		},
 		{
+			val: func(c *channel) float64 { return c.E2eLatency.Percentiles[0]["value"] },
+			vec: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "e2e_latency_99p",
+				Help:      "e2e latency 99th percentile",
+			}, labels),
+		},
```

Comment:
could you please consider adding the percentile information to a label "percentile". That's how most of the prometheus statistical metrics are represented